TYPO3.Flow/Users/maltemuth/src/vendor/neos/flow-development-3.2/Packages/Application/TYPO3.Flow/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "description": "Add description here",
+    "type": "neos-package",
+    "require": {
+        "typo3/flow": "*"
+    },
+    "name": "typo3/flow",
+    "autoload": {
+        "psr-0": {
+            "TYPO3\\Flow": "Classes/"
+        }
+    },
+    "extra": {
+        "neos": {
+            "package-key": "TYPO3.Flow"
+        }
+    }
+}
```

Comment:
Seems that accidentially sneaked in with 41656005ee9cf652165bd36f06b5aaceea5e89a6